css/emoji-mart.css

```diff
@@ -72,9 +72,11 @@
   max-width: 22px;
 }
 
-.emoji-mart-anchors svg {
+.emoji-mart-anchors svg,
+.emoji-mart-anchors img {
   fill: currentColor;
-  max-height: 18px;
+  height: 18px;
```

Comment:
Is the reason for just max-height before because some of the svgs aren't 18px ? because this would stretch any images to exactly 18px

Reply:
Not sure why they had just max-height and no max-width before but when we try to use max-height/max-width, the images that are less than 18px appear smaller and cause uneven category anchors so it appears to be better to just stretch the images so that all category anchors are the same size and consistent

Reply:
Looking at the history here: https://github.com/missive/emoji-mart/pull/77/files

We may want to check that this still works in IE.

Reply:
Just tested in IE 11 and this change doesn't seem to be causing any issues!